.travis.yml

```diff
@@ -1,28 +1,35 @@
 language: php
 
 php:
-    - 5.3
-    - 5.4
-    - 5.5
-    - 5.6
-    - hhvm
+  - 5.3
+  - 5.4
+  - 5.5
+  - 5.6
+  - hhvm
```

Comment:
Most of travis files have two spaces indent.

You can add a better diff by addin `?w=0` on diff url.

Let me know if you still want indentation back.